app/upw/templates/pdf-preview-and-declaration/components/pdf-template/template.njk

```diff
@@ -776,7 +776,8 @@
             ]
         }) }}
 
-        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+         <hr width="1220px;" color="black" size="2">
```

Comment:
I'm not keen on having values hard coded in to the template unless there's no way around it - are you able to add these params to a CSS class instead and then use that class for both the instances you've changed?

Reply:
Will go down the CSS route and try it out